test/extended/dr/static_pod_rollouts.go

```diff
@@ -0,0 +1,139 @@
+
+		isUnsupportedUnsafeEtcd, err := isUnsupportedUnsafeEtcd(&etcdCluster.Spec.StaticPodOperatorSpec)
+		o.Expect(err).ToNot(o.HaveOccurred())
+		o.Expect(isUnsupportedUnsafeEtcd).To(o.BeFalse(), "quorum guard is off, this will allow the CEO to act without quorum")
+
+		etcdPods, err := e2epod.GetPods(context.Background(), oc.AdminKubeClient(), etcdNamespace, map[string]string{"app": "etcd"})
+		o.Expect(err).ToNot(o.HaveOccurred())
+		initialEtcdPodCount = len(etcdPods)
+
+		masterNodes := masterNodes(oc)
+		etcdTargetNodeName = masterNodes[0].Name
+	})
+
+	g.AfterEach(func(ctx context.Context) {
+
+		// Reset the log level back to initial log level to leave the cluster in the same state.
+		g.GinkgoT().Log("resetting the log level back to intial log level")
+		data := fmt.Sprintf(`{"spec": {"logLevel": "%s"}}`, initialLogLevel)
+		_, err := oc.AdminOperatorClient().OperatorV1().Etcds().Patch(ctx, "cluster", types.MergePatchType, []byte(data), metav1.PatchOptions{})
+		o.Expect(err).ToNot(o.HaveOccurred())
+
+		g.GinkgoT().Log("debugging into the same node to add back the etcd static pod manifest")
+		err = oc.AsAdmin().Run("debug").Args("-n", etcdNamespace, "node/"+etcdTargetNodeName, "--", "chroot", "/host", "/bin/bash", "-c", "mv /var/lib/etcd-backup/etcd-pod.yaml /etc/kubernetes/manifests/").Execute()
+		o.Expect(err).ToNot(o.HaveOccurred(), "failed to add etcd static pod back to the node")
+
+		g.GinkgoT().Log("waiting for all the etcd instances to be available to return the cluster to its original state")
+		waitForReadyEtcdStaticPods(oc.AdminKubeClient(), initialEtcdPodCount)
+
+	})
+
+	g.It("is able to block the rollout of a revision when the quorum is not safe", func(ctx context.Context) {
+		var err error
+
+		g.GinkgoT().Log("debugging into the node to remove the etcd static pod manifest")
+		err = oc.AsAdmin().Run("debug").Args("-n", etcdNamespace, "node/"+etcdTargetNodeName, "--", "chroot", "/host", "/bin/bash", "-c", "mkdir -p /var/lib/etcd-backup && mv /etc/kubernetes/manifests/etcd-pod.yaml /var/lib/etcd-backup").Execute()
+		err = errors.Wrap(err, "failed to remove etcd static pod from the node")
+		o.Expect(err).ToNot(o.HaveOccurred())
+
+		g.GinkgoT().Log("waiting for the etcd pod to be removed")
+		waitForReadyEtcdStaticPods(oc.AdminKubeClient(), initialEtcdPodCount-1)
+
+		g.GinkgoT().Log("ensuring the quorum guard dependent controllers have degraded")
+		conditionTypes := []string{
+			"EtcdCertSignerControllerDegraded",
+			"EtcdEndpointsDegraded",
+			"TargetConfigControllerDegraded",
+		}
+		o.Expect(wait.PollUntilContextTimeout(ctx, 10*time.Second, 5*time.Minute, true, func(ctx context.Context) (done bool, err error) {
+			// retrieve the operator status
+			etcdCluster, err := oc.AdminOperatorClient().OperatorV1().Etcds().Get(ctx, "cluster", metav1.GetOptions{})
+			if err != nil {
+				klog.Infof("error while getting operator status: %v", err)
```

Comment:
This should be an error log no?
```suggestion
				klog.Errorf("error while getting operator status: %v", err)
```

Reply:
Yes. I have changed it to use `Errorf`. Was previously following the template used [here](https://github.com/openshift/cluster-etcd-operator/blob/164c37f68839b6d76bd724bb1ff9316bd62f6f97/test/e2e/backup_test.go#L375)